File: python/iniciante/uri1015.py
import math

list1 = input().split(" ")
list2 = input().split(" ")

x1, y1 = list1
x2, y2 = list2

distance = math.sqrt((float(x2) - float(x1))**2 + (float(y2) - float(y1))**2)

print ('{:.4f}'.format(distance))

File: javascript/iniciante/uri1015.js
let input = require('fs').readFileSync('/dev/stdin', 'utf8');
let lines = input.split('\n');

let line1 = lines.shift().split(' ');
let line2 = lines.shift().split(' ');

let x1 = Number(line1.shift());
let y1 = Number(line1.shift());
let x2 = Number(line2.shift());
let y2 = Number(line2.shift());

let distance = Math.sqrt(Math.pow((x2 - x1),2) + Math.pow((y2 - y1),2));

console.log(distance.toFixed(4));